yum-requirements.txt

```diff
@@ -11,36 +11,34 @@
 # Keep it sorted.
 autoconf
 bison
-build-essential
-clang-format
 curl
+devtoolset-7
 doxygen
+elfutils-libelf-devel
 flex
-g++
+gcc-c++
 git
-libelf1
-libelf-dev
-libftdi1-2
-libftdi1-dev
-libssl-dev
-libusb-1.0-0
-lsb-release
+libftdi-devel
+libusb-devel
+libxslt-devel
+# Need to run `scl enable llvm-toolset-7 bash` for below to take effect.
+llvm-toolset-7
 make
 # A requirement of the prebuilt clang toolchain.
 ncurses
 ninja-build
 openssl11-libs
 openssl11-devel
-pkgconf
+pkgconfig
 python3
 python3-pip
 python3-setuptools
 python3-wheel
 # The pip-installed version does not come with LibYAML support by default,
 # which significantly speeds up the parsing/dumping of YAML files.
-python3-yaml
+python3-pyyaml
+redhat-lsb-core
 srecord
 tree
-xsltproc
```

Comment:
You need to re-add this package, or building the documentation won't work.

Reply:
I assumed that this was now covered by `libxslt-devel`. But I might be wrong.